src/addQuestion/index.js

```diff
@@ -0,0 +1,94 @@
+import { useState } from "react";
+import { connect } from "react-redux";
+import { createQuestion } from "../store/question.js";
+import { questionFactory } from "../question.js";
+
+import "./index.css";
+
+let AddQuestionForm = ({ onSave, positionClass }) => {
+  const [question, setQuestion] = useState("");
+  const [askee, setAskee] = useState("");
+  const [status, setStatus] = useState("Unanswered");
+
+  const onChangeQuestion = ({ target: { value = "" } = {} } = {}) =>
+    setQuestion(value);
+  const onChangeAskee = ({ target: { value = "" } = {} } = {}) =>
+    setAskee(value);
+  const onChangeStatus = ({ target: { value = "" } = {} } = {}) =>
+    setStatus(value);
+
+  const onSubmit = (event) => {
+    event.preventDefault();
+    const newQuestion = questionFactory({

```

Comment:
`questionFactory` (and related logic) would not need to exist if your action creator can handle the defaults and you provide correct defaults to `useState`.